iOS/README.md

```diff
@@ -9,7 +9,20 @@
 
 Ping provides these iOS samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
 
-### Integrate with PingOne Advanced Identity Cloud / PingAM:
+The Ping SDK for iOS enables you to integrate your SPA with our services.
+There are two methods of UI integration:
```

Comment:
This section confuses me as it blends Embedded and Centralized terms too often in the same sentence